provision.yaml

```diff
@@ -4,7 +4,7 @@ default:
   images: ['waffleimage/centos7']
 waffle_deb:
   provisioner: docker
-  images: ['waffleimage/debian8', 'waffleimage/debian9', 'waffleimage/ubuntu16.04', 'waffleimage/ubuntu18.04']
+  images: ['waffleimage/debian8', 'waffleimage/debian9', 'waffleimage/ubuntu16.04', 'waffleimage/ubuntu16.04', 'waffleimage/ubuntu18.04']
```

Comment:
Hey @sheenaajay ,
Added ubuntu16 twice. Please update it to ubuntu14 ;)
cheers!

Reply:
Thanks @lionce .Changes are incorproated.